packages/dataviews/src/components/dataform/index.tsx

```diff
@@ -10,22 +10,38 @@ import type { DataFormProps } from '../../types';
 import { DataFormProvider } from '../dataform-context';
 import { normalizeFields } from '../../normalize-fields';
 import { DataFormLayout } from '../../dataforms-layouts/data-form-layout';
+import { MIXED_VALUE } from '../../constants';
 
 /**
  * Loops through the list of data items and returns an object with the intersecting ( same ) key and values.
+ * Skips keys that start with an underscore.
  *
  * @param data list of items.
  */
 function getIntersectingValues< Item extends object >( data: Item[] ): Item {
```

Comment:
I just noticed that this function is trying to combine the items into an `Item` object. Unfortunately, I don't think that's possible really because the `Symbol` is not a valid `Item` value and the nesting is not the right thing to do.

I think what we should be doing here is calling `field.getValue` for all the fields (maybe the fields that are within the form) and compute the result in a merged object (that is not `Item`) who keys are the field keys and values are the result of `getValue` calls.

This is necessary because we can't really assume/know the shape of valid `Item`. this is something we discussed IRL with @oandregal and @gigitux at Rome core days.

Reply:
Hmm yeah, I can give this approach a try. I agree it makes more sense, but will need quite a bit more changes I believe.
For example, if we do generate this new merged object using `getValue` and receive an array of `data` do we pass the array of data down to each field? or the merged object? or just the first item in the array ( probably the whole array, but currently a lot of fields pass that straight into `getValue`  ).

Reply:
I agree it's not very straightforward. Maybe the current PR can be a good interim solution (although not entirely "correct"). Should we try it in a separate PR to see how they compare?

Reply:
> Should we try it in a separate PR to see how they compare?

Yeah a different PR is a good idea, otherwise it may be hard to see the pros/cons of each approach.